rdkit/Chem/Randomize.py

```diff
@@ -8,64 +8,30 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
+import random
 from rdkit import Chem
-from rdkit import RDRandom as random
 
 
-def RandomizeMolBlock(molB):
-  splitB = molB.split('\n')
-  res = []
-  res.extend(splitB[0:3])
-  idx = 3
-  inL = splitB[idx]
-  res.append(inL)
-  nAts = int(inL[0:3])
-  nBonds = int(inL[3:6])
+def _shuffle(indices: list[int]) -> list[int]:
+    # Shuffle without in-place mutation.
+    # See https://docs.python.org/3/library/random.html#random.shuffle.
+    return random.sample(indices, len(indices))
 
-  idx += 1
-  atLines = splitB[idx:idx + nAts]
 
-  order = list(range(nAts))
-  random.shuffle(order, random=random.random)
+def RandomizeMolBlock(molblock: str) -> str:
+    mol = Chem.MolFromMolBlock(molblock, sanitize=False, removeHs=False)
+    atom_indices = [atom.GetIdx() for atom in mol.GetAtoms()]
+    atom_indices_randomized = _shuffle(atom_indices)
+    if len(atom_indices) > 1:
+        # Enforce randomization.
```

Comment:
Picky point: This isn't enforcing randomization, it's enforcing a different result.
The truly random approach would also sometimes return the same ordering.

I'm find leaving the code the way it is, but this comment should be removed/changed

Reply:
I agree, the comment is incorrect. See [2d07b45](https://github.com/rdkit/rdkit/pull/7232/commits/2d07b45f4b414983e042feaf0c006d174ef2d84a).